vochain/vochaininfo/vochaininfo.go

```diff
@@ -14,6 +14,10 @@ import (
 	"go.vocdoni.io/dvote/vochain/state"
 )
 
+const (
+	defaultBlockTime = 10000 // 10 seconds
```

Comment:
why is this different from the other defaultBlockTime (12s)?

Reply:
Actually we should unity this into tye types package